app/containers/markdown/new/NewMarkdown.stories.tsx

```diff
@@ -423,10 +423,73 @@ const markdownLink = [
 	}
 ];
 
+const markdownLinkEmphatic = [
+	{
+		type: 'PARAGRAPH',
+		value: [
+			{
+				type: 'LINK',
+				value: {
+					src: {
+						type: 'PLAIN_TEXT',
+						value: 'https://rocket.chat/'
+					},
+					label: [
+						{
+							type: 'PLAIN_TEXT',
+							value: 'Normal Link - '
+						},
+						{
+							type: 'BOLD',
+							value: [
+								{
+									type: 'PLAIN_TEXT',
+									value: 'Bold'
+								}
+							]
+						},
+						{
+							type: 'PLAIN_TEXT',
+							value: ' '
+						},
+						{
+							type: 'STRIKE',
+							value: [
+								{
+									type: 'PLAIN_TEXT',
+									value: 'strike'
+								}
+							]
+						},
+						{
+							type: 'PLAIN_TEXT',
+							value: ' and '
+						},
+						{
+							type: 'ITALIC',
+							value: [
+								{
+									type: 'PLAIN_TEXT',
+									value: 'Italic'
+								}
+							]
+						},
+						{
+							type: 'PLAIN_TEXT',
+							value: ' Styles'
+						}
+					]
+				}
+			}
+		]
+	}
+];
+
 export const Links = () => (
 	<View style={styles.container}>
 		<NewMarkdown tokens={rocketChatLink} />
 		<NewMarkdown tokens={markdownLink} />
+		<NewMarkdown tokens={markdownLinkEmphatic} />
```

Comment:
This is a false cognate on english.
The way you wrote, the link is empathetic/emphatic means the link can understand how someone else feels 🤔 
And you meant to write is that the link contains emphasis elements.
Make sure you review other places on this PR as well.